pom.xml

```diff
@@ -392,6 +388,16 @@
           <artifactId>gmavenplus-plugin</artifactId>
           <version>2.1.0</version>
         </plugin>
+        <plugin>
+          <groupId>org.jacoco</groupId>
+          <artifactId>jacoco-maven-plugin</artifactId>
```

Comment:
Defined version for `jacoco-maven-plugin` in parent pom to eliminate warnings like the following:

```
[WARNING] Some problems were encountered while building the effective model for io.kubernetes:client-java:bundle:18.0.0-SNAPSHOT
[WARNING] 'build.plugins.plugin.version' for org.jacoco:jacoco-maven-plugin is missing. @ line 123, column 21
[WARNING] 
[WARNING] Some problems were encountered while building the effective model for io.kubernetes:client-java-spring-integration:bundle:18.0.0-SNAPSHOT
[WARNING] 'build.plugins.plugin.version' for org.jacoco:jacoco-maven-plugin is missing. @ line 77, column 15
[WARNING] 
[WARNING] Some problems were encountered while building the effective model for io.kubernetes:client-java-spring-aot-integration:bundle:18.0.0-SNAPSHOT
[WARNING] 'build.plugins.plugin.version' for org.jacoco:jacoco-maven-plugin is missing. @ line 52, column 21
[WARNING] 
[WARNING] It is highly recommended to fix these problems because they threaten the stability of your build.
[WARNING] 
[WARNING] For this reason, future Maven versions might no longer support building such malformed projects.
```

I believe this issue was introduced in https://github.com/kubernetes-client/java/pull/2493 when the `jacoco-maven-plugin` dependency was moved to the parent pom, but not added to the `pluginManagement` block.